docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1047,6 +1047,20 @@ How to **investigate**:
   1. The alert fired because of a bug in Mimir: fix it.
   1. The alert fired because of a bug or edge case in the continuous test tool, causing a false positive: fix it.
 
+### MimirRingMembersMismatch
+
+This alert fires when the number of ring members does not match the number of running replicas.
+
+How it **works**:
+
+- The alert compares each component (currently just `ingester`) against the number of `up` instances for the component in that namespace/cluster.
+
+How to **investigate**:
+
+- Check the [hash ring web page]({{< relref "../reference-http-api/index.md#ingesters-ring-status" >}}) for the component for which the alert has fired, and look for unexpected instances in the list.
```

Comment:
```suggestion
- For the component for which the alert has fired, and to look for unexpected instances in the list, see [Ingesters ring status]({{< relref "../reference-http-api/index.md#ingesters-ring-status" >}}).
```

Reply:
I'm not sure this suggestion makes sense. This is an action to take once you know a component that is mis-matched.